sdk/loadtesting/load-testing-rest/samples-dev/startTestProfileRun.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to run a test profile and get recommendations from it.
+ *
+ * @summary Demonstrates how to start a test profile run.
+ */
+
+import AzureLoadTesting, { isUnexpected, getLongRunningPoller } from "@azure-rest/load-testing";
+import { DefaultAzureCredential } from "@azure/identity";
+import { randomUUID } from "node:crypto";
+
+async function main(): Promise<void> {
+  /**
+   * The dataplane endpoint for the Azure Load Testing resource.
+   * Refer to https://learn.microsoft.com/rest/api/loadtesting/data-plane-uri to understand how to obtain the data-plane endpoint.
+   */
+  const endpoint = process.env["LOADTESTSERVICE_ENDPOINT"] || "";
+
+  /** Microsoft Entra ID authentication */
+  /**
+   * In this sample you can populate the three AZURE_CLIENT_ID, AZURE_CLIENT_SECRET & AZURE_TENANT_ID variables for Microsoft Entra ID auth
+   */
+  const credential = new DefaultAzureCredential();
+  const testProfileId = process.env["LOADTESTSERVICE_TESTPROFILEID"] || ""; // TestProfileId of a test profile already created.
+
+  const testProfileRunId = randomUUID(); // Random ID for the Test Profile Run
+  const displayName = "Sample Test Profile Run";
+
+  // Build a client through AAD
+  const client = AzureLoadTesting(endpoint, credential);
+
+  // Creating the test profile run
+  const testProfileRunCreationResult = await client
+    .path("/test-profile-runs/{testProfileRunId}", testProfileRunId)
+    .patch({
+      contentType: "application/merge-patch+json",
+      body: {
```

Comment:
same, is it something like we will copy path parameters to the body parameters in the SDK itself ?

Reply:
Yes, it does